### ISSUE TITLE
Nike Store

### ISSUE THREAD

**Issue by azafrak** [StevenBlack/hosts#2242]
#### What adblocker/DNS cloud service are you using?
NextDNS

#### Which blocklists are used?

Steven Black

#### Which domain(s) should be unblocked?

cdn.optimizely.com

#### Why should the domain(s) be unblocked? If necessary, please describe the steps to reproduce.

In order to reach the payment page of the product added to the cart in the nike store, I need to add this cdn service to the white list. Otherwise, the white page is waiting and the payment screen does not appear. You can add a product to the cart and test it yourself.

### Attacment
![Brahmancow](https://user-images.githubusercontent.com/30311445/220626285-d994d27d-490c-4c53-8aea-b7299b161c03.png)
![Waterdogs](https://user-images.githubusercontent.com/30311445/220626351-6afde0e6-aed7-46f0-ad57-dd17f21ca6a0.png)
![image](https://user-images.githubusercontent.com/30311445/220626503-2b1efe05-3699-4a96-9b32-960ca3f828aa.png)

**Comment by icedterminal** [comment]
This is a regional issue. US and Canada do not experience this as `nike.com/checkout` is used for the entire process.

The domain you are requesting to be removed actively includes tracking based on how it is implemented by the company using it. It can employ basic content delivery or dynamic A/B testing mechanisms that track end users (stats/metrics, etc.)

> Optimizely Web Experimentation uses persistent visitor-level cookies and localStorage to uniquely identify visitors, track and attribute their actions to experiments and personalization campaigns, and deliver consistent experiences across page loads and visitor sessions.

https://support.optimizely.com/hc/en-us/articles/4410284274573-Intelligent-tracking-prevention-and-Optimizely-Web-Experimentation

The services does offer a way for users for to opt-out of tracking where legislation requires. However, for those that do not, it can be forced upon you by default. Users would have to know the site they are visiting uses the CDN with tracking and would need to deny the cookie in the browser setting. 

https://support.optimizely.com/hc/en-us/articles/4410284177677-Enable-opt-in-options-for-Optimizely-Experimentation-cookies-and-local-storage

As such, the domain is added to the host based on principle. If you find it has broken as service, you can whitelist it on your end.

**Comment by StevenBlack** [comment]
Hi @azafrak thank you for this.

This is a duplicate of #2087 and sources and reasoning for carrying the domain `cdn.optimizely.com` is found there.

Closing.
